components/script/dom/abortsignal.rs

```diff
@@ -208,6 +281,17 @@ impl AbortSignalMethods<crate::DomTypeHolder> for AbortSignal {
         signal
     }
 
+    /// <https://dom.spec.whatwg.org/#dom-abortsignal-any>
+    fn Any(
+        global: &GlobalScope,
+        signals: Vec<DomRoot<AbortSignal>>,
+        can_gc: CanGc,
+    ) -> DomRoot<AbortSignal> {
+        // The static any(signals) method steps are to return the result
+        // of creating a dependent abort signal from signals using AbortSignal and the current realm.
+        Self::create_dependent_abort_signal(signals, global, can_gc)
```

Comment:
Thanks for implementing `Any`. I was working on this in #36936, so it would be good to check to avoid duplicate work.

Reply:
Ah I asked @jdm if you folks weren't working on it, as I indeed wanted to avoid duplicate work. He said that you weren't, but I wasn't aware that you were continuing on it in a different ticket.